### PR TITLE
Fix/847 key figures

### PR DESCRIPTION
**Pull request by necoline**

### CODE REVIEW

**Comment by necoline** [comment]
## Adds Key Figures to Region and Country Pages
## Description
- Replaces Homestats component and container with one reusable `key-figures-header-component`
- Creates a reusable tooltip
- Creates a icon component 
- Updates style to match mockups and create continuity across pages
- Removes Country Stats -- this need to be verified as a desired outcome. I deleted this because there is no data currently coming through on theses stats and they don't exist on the updated designs.

## Related Issue
#847 

## Motivation and Context
- The homestats container applied a brittle tooltip that applied styles from a map popover. The values were hardcoded in and essentially un-extractable from the homestats
- the homestats component hardcoded the key figures and was nested inside the container that was written specifically for the use of the tooltip
- I brought in a react tooltip and updated the styles to match the mockups
- I created a single reusable key figures component and dynamically added values into the component.

## Screenshots (if appropriate):
Home page:
image
 Home page with tooltip
image

Region page:
image

Country page:
image

Homepage fullscreen:
image

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code is readable and well commented.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

File: app/assets/scripts/components/common/key-figures-header.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import c from 'classnames';
+import { percent, shortenLargeNumber } from '../../utils/format';
+import BlockLoading from '../block-loading';
+import { environment } from '../../config';
+import { PropTypes as T } from 'prop-types';
+import Tooltip from './tooltip';
```
Comment (batpad) [inline app/assets/scripts/components/common/key-figures-header.js]: The file looks to be called `./Tooltip` with a capital T?

**Comment by batpad** [comment]
@necoline I seem to be getting some errors when trying to run this:

```
Javascript error: { Error: Cannot find module './tooltip' from '/home/sanj/seed/go-frontend/app/assets/scripts/components/common'
    at /home/sanj/seed/go-frontend/node_modules/browser-resolve/node_modules/resolve/lib/async.js:55:21
    at load (/home/sanj/seed/go-frontend/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/home/sanj/seed/go-frontend/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /home/sanj/seed/go-frontend/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:152:21)
  stream: 
   Labeled {
     _readableState: 
      ReadableState {
        objectMode: true,
        highWaterMark: 16,
        buffer: BufferList { length: 0 },
        length: 0,
        pipes: [Object],
        pipesCount: 1,
        flowing: true,
        ended: false,
        endEmitted: false,
        reading: true,
        sync: false,
        needReadable: true,
        emittedReadable: false,
        readableListening: false,
        resumeScheduled: false,
        destroyed: false,
        defaultEncoding: 'utf8',
        awaitDrain: 0,
        readingMore: false,
        decoder: null,
        encoding: null },
     readable: true,
     domain: null,
     _events: 
      { end: [Array],
        error: [Function],
        data: [Function: ondata],
        _mutate: [Object] },
     _eventsCount: 4,
     _maxListeners: undefined,
     _writableState: 
      WritableState {
        objectMode: true,
        highWaterMark: 16,
        finalCalled: false,
        needDrain: false,
        ending: true,
        ended: true,
        finished: true,
        destroyed: false,
        decodeStrings: true,
        defaultEncoding: 'utf8',
        length: 0,
        writing: false,
        corked: 0,
        sync: false,
        bufferProcessing: false,
        onwrite: [Function],
        writecb: null,
        writelen: 0,
        bufferedRequest: null,
        lastBufferedRequest: null,
        pendingcb: 0,
        prefinished: true,
        errorEmitted: false,
        bufferedRequestCount: 0,
        corkedRequestsFree: [Object] },
     writable: false,
     allowHalfOpen: true,
     _options: { objectMode: true },
     _wrapOptions: { objectMode: true },
     _streams: [ [Object], [Object] ],
     length: 2,
     label: 'deps' } }
```

This is when building / running `yarn serve` - maybe due to what looks like a typo that I commented on above? Let me know when I should test again.

**Comment by necoline** [comment]
that's weird, I can't seem to reproduce this, the file should be lowercase `tooltip.js` and imported as `import Tooltip from './tooltip';` I made that change to the file name, which will hopefully resolve this error. I also spotted a but with the tooltip ids that I fixed as well

**Comment by batpad** [comment]
@necoline this is looking good to me and seems to be working well.

Just minor styling: It feels to me like the spacing between the boxes on the country / region page might be a bit too much and it maybe nicer to reduce the space between the boxes when there are less. However, this is a bit of an opinion, and I'm also fine to merge this and get styling feedback.

**Pull request merged**